docs/src/components/banner/AppFrameBanner.tsx

```diff
@@ -14,7 +14,11 @@ export default function AppFrameBanner() {
   // eslint-disable-next-line react-hooks/rules-of-hooks -- FEATURE_TOGGLE never changes
   const pageContext = React.useContext(PageContext);
   const productName = convertProductIdToName(pageContext) || 'MUI';
-  const message = `Influence ${productName}'s 2024 roadmap! Participate in the latest Developer Survey`;
+  const surveyMessage = `Influence ${productName}'s 2024 roadmap! Participate in the latest Developer Survey`;
+  const blackFridayMessage = `Black Friday is here! Don't miss out on the best offers of the year.`;
```

Comment:
This will be challenging to remove once it propagates to MUI X, Toolpad, etc at the right time. Not 100% convinced that we should have this. The change itself makes sense, e.g. 
<img width="787" alt="SCR-20241127-mrgx" src="https://github.com/user-attachments/assets/88fe6b08-083f-4099-afb5-24d7c8d40887">

But it's work 😄. #44566 is great though.

Can we reopen a new PR to make this display time sensitive?

Reply:
@mnajdova I have added this instance under https://www.notion.so/mui-org/React-Engineer-Docs-infra-02023ab7104241888ca2ab3bcb32e19d?pvs=4#14bcbfe7b66080bf8f9ad8a17b8cc2d3. Vadym couldn't have known how our docs-infra is distributed, Victor can't be accountable for engineering constraints. So at the end of the days, it feels like this role gap.

Reply:
Handling this in #44630